bors/src/git.rs

```diff
@@ -217,6 +217,9 @@ impl Git {
         } else {
             std::env::current_dir().unwrap().join(ssh_key_file)
         };
+        if !path.is_file() {
```

Comment:
This is [not really recommend way](https://doc.rust-lang.org/nightly/std/fs/struct.FileType.html#method.is_file) to check for a regular file. 
Use File::open and check its result to know if one can read/write to it.